src/Api/Serializer/UserSerializer.php

```diff
@@ -36,7 +36,7 @@ protected function getDefaultAttributes($user)
 
         $gate = $this->gate->forUser($this->actor);
 
-        $canEdit = $gate->allows('edit', $user);
+        $canEdit = $gate->allows('edit.credentials') || $gate->allows('edit.groups') || $gate->allows('edit.username') ? true : false;
```

Comment:
This will hopefully be removed in the refactor, however the frontend uses the `canEdit` json attribute to determine if the edit option should appear in the dropdown for user editing. Unless you think that splitting that out makes more sense?